assets/app/views/directives/events.html

```diff
@@ -0,0 +1,36 @@
+<div class="middle-content">
```

Comment:
Hm, not sure you should have this here. The `middle-content` div should be in the parent page?